docs/sparkr.md

```diff
@@ -210,6 +210,43 @@ head(df)
 {% endhighlight %}
 </div>
 
+### Model Formulae
+
+SparkR allows the fitting of generalized linear models over DataFrames using the glm() function. Under the hood, SparkR uses MLlib to train a model of the specified family. In Spark 1.5, we support a subset of the available R formula operators for model fitting, including '~', '.', '+', and '-'. The example below shows the use of R formulae with SparkR.
```

Comment:
* Remove `In Spark 1.5, `. 
* Link `glm()` to the generated html doc, e.g. `api/R/glm.html`.